traffic_portal/app/src/common/modules/table/topologyCacheGroupServers/table.topologyCacheGroupServers.tpl.html

```diff
@@ -0,0 +1,48 @@
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+
+
+<div class="modal-header">
+    <button type="button" class="close" ng-click="cancel()"><span aria-hidden="true">&times;</span><span class="sr-only">Close</span></button>
+    <h3 class="modal-title">Servers assigned to {{::cacheGroupName}}</h3>
+</div>
+<div class="modal-body">
+    <table id="topologyCacheGroupServersTable" class="table responsive-utilities jambo_table" style="table-layout:fixed; width:100%;">
+        <thead>
+        <tr class="headings">
+            <th>Host</th>
+            <th>Domain</th>
+            <th>CDN</th>
+            <th>Status</th>
+        </tr>
+        </thead>
+        <tbody>
+        <tr ng-repeat="s in ::cacheGroupServers">
+            <td data-search="^{{::s.hostName}}$">{{::s.hostName}}</td>
+            <td data-search="^{{::s.domainName}}$">{{::s.domainName}}</td>
+            <td data-search="^{{::s.cdnName}}$">{{::s.cdnName}}</td>
+            <td data-search="^{{::s.status}}$">{{::s.status}}</td>
+        </tr>
+        </tbody>
+    </table>
+</div>
+<div class="modal-footer">
+    <button class="btn btn-primary" ng-click="cancel()">Close</button>
+</div>
```

Comment:
If you want a footer you could use a [`<footer>` tag](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/footer)

Reply:
then i gotta mess with the css i'm pretty sure i'm getting for free from bootstrap....maybe?

Reply:
dunno, but I'd bet the selector just looks for class

Reply:
no, it busted the css. guess `<div class="modal-footer">` will have to do.

Reply:
That appears to be because in our theme CSS we're setting all `footer` elements to have a left-margin of 230px. I'm not gonna ask you to change it or anything, but it seems like a bad idea to apply a global margin to all elements of a certain tag name - it'd be one thing if it was in a component's styling but idk about globally. So just something of which we should be aware.